### PR TITLE
FIB Playbook and, BGP rules modification

### PR DESCRIPTION
**Pull request by gvarao**
- Added forwarding tables route summary playbook
- Enhanced route summary(rib) playbook with static threshold according to Javier's request
- Added new bgp advertised route count check rule
- Delete two bgp rules which are not required as addressed functionality in existing rules i.e dynamic thresholds

### CODE REVIEW

File: juniper_official/Protocols/Fib/fib-summary.rule [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                 }
+             }
+            /*
+             * Fields defined with dynamic threshold ML algorithms
+             */
+             field dt-route-count {
+                 formula {
+                     dynamic-threshold {
+                         algorithm k-means;
+                         learning-period 1m;
```
Comment (ConstrainedRandomVar) [inline juniper_official/Protocols/Fib/fib-summary.rule]: Given that our default retention policy is 7 days, this combination of pattern-periodicity and learning-period won't work. Can we bump down the pattern-periodicity to 1d or 1d1h and the learning period to 7 days?

Also, can you please update to the new syntax and not use the deprecated syntax. Though, I am not sure whether the code is in yet. :(
But, can you check with Mohit on what he plans to introduce?

File: juniper_official/Protocols/Rib/route-summary.playbook [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*
- * Playbook contains multiple rules which checks the statistics of BGP neighbor
- * sessions and notifies when anomalies are found.
+ * Playbook contains multiple rules which monitors each route-table's and 
+ * protocol's router count and notifies when anomalies are found.
  *
  * 1) Rule "check-ascertain-routes" sets the dynamic thresholds based on 
  *    the learned data (route count of each table) using MI (Machine Learning)
```
Comment (ConstrainedRandomVar) [inline juniper_official/Protocols/Rib/route-summary.playbook]: Presumably, MI refers to ML?

Comment (gvarao) [reply juniper_official/Protocols/Rib/route-summary.playbook]: Modified and committed code

File: juniper_official/Protocols/Fib/fib-summary.rule (hunk 1 above)
Comment (gvarao) [reply juniper_official/Protocols/Fib/fib-summary.rule]: Modified and committed code

File: juniper_official/Protocols/Fib/fib-summary.rule [hunk 3]
```diff
@@ -0,0 +1,162 @@
+/*
```
[the remark this answers is not in the document]
Comment (gvarao) [reply juniper_official/Protocols/Fib/fib-summary.rule]: Modified and committed code

File: juniper_official/Protocols/Fib/fib-summary.rule (hunk 1 above)
Comment (gvarao) [reply juniper_official/Protocols/Fib/fib-summary.rule]: Modified and committed code

File: juniper_official/Protocols/Rib/route-summary.playbook (hunk 2 above)
Comment (gvarao) [reply juniper_official/Protocols/Rib/route-summary.playbook]: Modified and committed code

**Pull request merged**